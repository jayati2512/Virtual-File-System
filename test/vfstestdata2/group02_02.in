createvfs school
createvfs school 20
mountvfs school
makedir /school teacher
makedir /school student
makedir /school 
makedir /school subject
addfile /school/subject math.txt /usr/include/syslog.h
addfile /school/subject english.txt /usr/include/eicode.h
addfile /school/student abc.txt /usr/include/ustat.h
addfile /school/student def.txt /usr/include/wait.h
addfile /school/teacher tone.txt /usr/include/ustat.h
addfile /school/teacher ttwo.txt /usr/include/wait.h
deletedir /school
deletedir /school/student
removefile /school/student/def.txt
removefile /school
removefile /school/student/abc.txt
deletedir /school/student
unmountvfs school
