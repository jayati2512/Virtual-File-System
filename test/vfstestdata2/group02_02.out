createvfs_FAILURE VFS_INSUFFICIENT_ARGUMENTS
createvfs_SUCCESS
mountvfs_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_FAILURE VFS_INSUFFICIENT_ARGUMENTS
makedir_SUCCESS
addfile_SUCCESS
addfile_SUCCESS
addfile_SUCCESS
addfile_SUCCESS
addfile_SUCCESS
addfile_SUCCESS
deletedir_FAILURE DIRECTORY_IS_NOT_EMPTY
deletedir_FAILURE DIRECTORY_IS_NOT_EMPTY
removefile_SUCCESS
removefile_FAILURE CANNOT_FIND_SPECIFIED_FILE
removefile_SUCCESS
deletedir_SUCCESS
unmountvfs_SUCCESS
